intel-sgx/enclave-runner/src/loader.rs

```diff
@@ -332,21 +342,26 @@ impl<'a> EnclaveBuilder<'a> {
     }
 
     pub fn build<T: Load>(mut self, loader: &mut T) -> Result<Command, anyhow::Error> {
+        let num_cpus = num_cpus::get();
+        let num_worker_threads = self.num_worker_threads.unwrap_or(num_cpus);
+        anyhow::ensure!(num_worker_threads > 0, "`num_worker_threads` cannot be zero");
+        anyhow::ensure!(num_worker_threads <= num_cpus, "`num_worker_threads` cannot exceed number of logical cores (`num_cpus::get()`)");
```

Comment:
I'm not sure this is a useful check, in the presence of CPU hotplug. The system could have more CPUs added to it later, ot it could have CPUs removed from it at run time. So we could disallow a number of worker threads that could later become legal, or we might allow a number of threads now that becomes illegal later.

Reply:
Hmm, I can remove the upper bound check in that case, or have a fixed upper bound (maybe something like 512?). Which one would you prefer?

Reply:
I think a fixed upper bound is fine, but 512 is too low. There are already systems with something like 320 logical cores, so we're not that far away from systems with 512. Something like 64K could be reasonable.

Reply:
Will make it 64K!